packages/exports/src/images/latexToSvg.ts

```diff
@@ -0,0 +1,44 @@
+import { liteAdaptor } from "mathjax-full/js/adaptors/liteAdaptor";
+import type { LiteAdaptor } from "mathjax-full/js/adaptors/liteAdaptor";
+import { RegisterHTMLHandler } from "mathjax-full/js/handlers/html";
+import { TeX } from "mathjax-full/js/input/tex";
+import { mathjax } from "mathjax-full/js/mathjax";
+import { SVG } from "mathjax-full/js/output/svg";
+
+let adaptor: LiteAdaptor | null = null;
+let document: ReturnType<typeof mathjax.document> | null = null;
+
+// Following mathjax direct pattern: https://github.com/mathjax/MathJax-demos-node/blob/master/direct/tex2svg
+
+function ensureInitialized() {
+  if (adaptor && document) {
+    return;
+  }
+
+  adaptor = liteAdaptor();
+  RegisterHTMLHandler(adaptor);
+
+  const tex = new TeX({
+    packages: [
+      "base",
+      "ams",
+      "noundefined",
+      "newcommand",
+      "boldsymbol",
+      "unicode",
+    ],
+  });
+
+  const svg = new SVG({
+    fontCache: "local",
+  });
+
+  document = mathjax.document("", { InputJax: tex, OutputJax: svg });
+}
+
+export function latexToSvg(latex: string, isDisplay = false): string {
+  ensureInitialized();
```

Comment:
Personal preference would be for this function to return (document, adaptor), then no need to assert not-null below

Reply:
You know, I did have that in an earlier version. Will put it back!